browser/ephemeral_storage/ephemeral_storage_browsertest.cc

```diff
@@ -100,6 +105,10 @@ class EphemeralStorageBrowserTest : public InProcessBrowserTest {
         https_server_.GetURL("b.com", "/ephemeral_storage.html");
     c_site_ephemeral_storage_url_ =
         https_server_.GetURL("c.com", "/ephemeral_storage.html");
+
+    ephemeral_storage::EphemeralStorageTabHelper::
+        SetKeepAliveTimeDelayForTesting(
```

Comment:
why not `ScopedFeatureList::InitAndEnableFeatureWithParameters`?

Reply:
is that still ok to use when the feature is enabled by default? If so I'll remove `SetKeepAliveTimeDelayForTesting`

Reply:
I guess it must be ok because it still has InitAndEnable for kBraveEphemeralStorage

Reply:
actually this seems to complicate running in both the enabled and disabled state so I'm going to leave it as-is

Reply:
that would still require a method on `EphemeralStorageTabHelper` to set the actual delay so not sure that's worth the extra effort since we always want the short delay in testing

Reply:
going with this instead https://github.com/brave/brave-core/pull/8526#discussion_r618306557

Reply:
I'm going to look at this in a follow-up so I can get this merged